x-pack/plugins/maps/public/render_app.tsx

```diff
@@ -96,7 +96,7 @@ export async function renderApp(
     }
 
     return (
-      <SharedUxServicesProvider {...getSharedUXPluginContext().getContextServices()}>
+      <ExitFullScreenButtonKibanaProvider coreStart={getCore()}>
```

Comment:
Should ExitFullScreenButtonKibanaProvider to moved closer to where ExitFullScreenButton is used? Does it still make sense to wrap at such a high level of the application?

Reply:
The reason I didn't is because if maps ends up using other Shared UX components, we'll need to combine providers with a utility I'm going to be writing next.  I can move it down, if you'd prefer.

Reply:
Its fine to leave where it is if there are future changes that make sense having it closer to the root react tree.

Reply:
Could you approve the changes, @nreese ?